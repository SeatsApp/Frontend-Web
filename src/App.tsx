import { Routes, Route } from "react-router-dom";
import Header from "./appbar/Header";
import { Box, Container, CssBaseline, ThemeProvider } from "@mui/material";
import { theme } from "./theme";
import Dashboard from "./dashboard/components/Dashboard";
import React from "react";
import UserBoard from "./userboard/components/UserBoard";
import { ManageBuildings } from "./buildings/components/ManageBuildings";
import { Floorplan } from "./floorplan/components/Floorplan";
import { CreateBuilding } from "./create_building/components/CreateBuilding";

function App() {
    return (
        <ThemeProvider theme={theme}>
            <CssBaseline />
            <Container>
                <Header />
                <Box style={{ marginTop: '100px' }}>
                    <Routes>
                        <Route path="/" element={<Dashboard />}/>
                        <Route path="/users" element={<UserBoard />}/>
                        <Route path="/manageBuildings" element={<ManageBuildings />} />
                        <Route path="/buildings/:buildingId" element={<CreateBuilding />} />
                        <Route path="/buildings/:buildingId/floors/:floorId/floorplan" element={<Floorplan />} />
                    </Routes>
                </Box>
            </Container>
        </ThemeProvider>
    );
}

export default App;
