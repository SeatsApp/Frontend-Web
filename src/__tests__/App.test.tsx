import React from 'react';
import TestRenderer from 'react-test-renderer';
import App from '../App';

test('renders learn react link', () => {
  const tree = TestRenderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});
