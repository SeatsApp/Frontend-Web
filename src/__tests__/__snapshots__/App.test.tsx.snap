// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
>
  Hallo
</p>
`;
